src/main/java/teammates/logic/core/CoursesLogic.java

```diff
@@ -668,23 +668,16 @@ public void deleteAllCoursesCascade(List<InstructorAttributes> instructorList) {
      * Moves a course to Recycle Bin by its given corresponding ID.
      * @return Soft-deletion time of the course.
```

Comment:
So in logic, we should talk about recycle bin rather than soft-deletion.

`@return the time when the course is moved to the recycle bin`